scripts/shell.sh

```diff
@@ -58,7 +69,17 @@ fi
 preflightCheck() {
   require_util curl "download dependencies"
   require_util tar "decompress nix installation package"
-  require_util unshare "verify kernel support for user namespaces"
+  case "$(uname)" in
+    Darwin)
+      require_util nix "make the magic happen. You are running on OSX, please make sure Nix is installed by following https://nixos.org/download.html#nix-install-macos"
```

Comment:
Good point, I'll add it now.

Reply:
I've put the command to paste and run actually, with a note that we need `--no-daemon`.

Reply:
Damn, `--no-daemon` installs are no longer supported on OSX

Reply:
Well, that is a bummer.
I will try to figure out an easy to maintain, way of keeping nix.conf in sync - for now this is not an issue at all, but if we start playing with some more advanced use cases it might be.